global-news-app/src/Charts/ChartsSection/PihChart/Legend/Legend.style.tsx

```diff
@@ -0,0 +1,29 @@
+import { scrollbarStyle } from "assets/styles/shared";
+import styled from "styled-components";
+
+export const LegendContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 11px;
+  padding: 0px 15px 0px;
```

Comment:
padding: 0px 16px 0px;
gap: 12px;
usually the designers use the numbers that are divided by 8
and use the spacing const instead the numbers